app/assets/stylesheets/govuk_publishing_components/components/_single-page-notification-button.scss

```diff
@@ -0,0 +1,32 @@
+.gem-c-single-page-notification-button__submit {
+  position: relative;
+  padding: govuk-spacing(2);
+  text-decoration: none;
+  margin: govuk-spacing(0);
+  box-shadow: inset 0 0 0 1px transparent;
+  border: 1px solid $govuk-border-colour;
+  color: $govuk-link-colour;
+  cursor: pointer;
+  background: none;
+
+  &:active {
+    top: 2px;
+  }
+
+  &:focus {
```

Comment:
that's bizarre! I can't replicate it. Can I ask which browser this is?

Reply:
Normal Chrome, presumably the same version as you. Interestingly I can't reproduce it either anymore, so that's good. Maybe it was fixed by one of the other unrelated changes.